assets/lokomotive-kubernetes/bootkube/variables.tf

```diff
@@ -162,3 +162,8 @@ variable "blocked_metadata_cidrs" {
   type        = list(string)
   default     = []
 }
+
+variable "bootstrap_tokens" {
+  description = "List of boostrap tokens for all the nodes in the cluster in the form token-id and token-secret."
+  type        = list(any)
```

Comment:
Why not `list(object(...))` ? Then mapping is not needed when passed to the template